endoreg_db/services/pseudonym_service.py

```
@@ -56,12 +56,12 @@ def generate_patient_pseudonym(patient: Patient) -> Tuple[str, bool]:
             patient.patient_hash = patient_hash
             patient.save(update_fields=['patient_hash'])
         
-        logger.info("Generated and persisted pseudonym for patient %s", patient.pk)
+        logger.info(f"Generated and persisted pseudonym for patient {patient.id}: {patient_hash[:8]}...")
         
         return patient_hash, True
```

Ruff (0.14.0)

Consider moving this statement to an `else` block

(TRY300)